BugsnagReactNative.podspec

```diff
@@ -23,9 +23,7 @@ Pod::Spec.new do |s|
                    'cocoa/vendor/kscrash/Source/KSCrash/**/*.{h,m,mm,cpp,c}',
                    'cocoa/vendor/kscrash/Source/Framework/**/*.{h,m,mm,cpp,c}',
 
-  s.public_header_files = 'cocoa/vendor/bugsnag-cocoa/Source/Bugsnag{,MetaData,Configuration,Breadcrumb,CrashReport}.h',
-                          'cocoa/vendor/bugsnag-cocoa/Source/BSGKSCrashReportWriter.h',
-                          'cocoa/BugsnagReactNative.h'
+  s.public_header_files = 'cocoa/**/{Bugsnag,BugsnagMetaData,BugsnagConfiguration,BugsnagBreadcrumb,BugsnagCrashReport,BSGKSCrashReportWriter}.h'
```

Comment:
Whoops. Good catch.

Although I'm able to build locally, my CI is failing so this story might not be over yet. I'll post once I've got this fully working with the podspec in this PR.